contracts/example/tests/test.rs

```diff
@@ -139,15 +136,27 @@ fn gmp_example() {
 
     // Setup source Axelar gateway
     let source_chain = String::from_str(&env, SOURCE_CHAIN_NAME);
-    let source_test_config = setup_app(&env, &source_chain);
+    let TestConfig {
+        signers: _,
+        gateway_client: _,
+        gas_service_client: source_gas_service_client,
+        its_client: _,
+        app: source_app,
```

Comment:
```suggestion
        gas_service_client: source_gas_service_client,
        app: source_app,
        ..
```
don't need to mention all the fields